components/playlist/resources/components/app.tsx

```diff
@@ -127,9 +135,29 @@ export class PlaylistPage extends React.Component<Props, State> {
     document.getElementById('player')?.setAttribute('src', this.getMediaSrc(itemId))
   }
 
+  createPlaylist = (playlist: PlaylistMojo.Playlist) => {
+    getPlaylistAPI().createPlaylist(playlist)
+  }
+
+  selectPlaylist = (playlistId: string) => {
+    const playlist = this.findPlaylistWithId(playlistId)
+    if (!playlist) return
+    getAllActions().selectPlaylist(playlist)
+  }
+
+  removePlaylist = (playlistId: string) => {
+    getPlaylistAPI().removePlaylist(playlistId)
+  }
+
   render () {
+    console.log(this.getCurrentPlaylist()?.name)
     return (
       <div id='playlistPage'>
+        <PlaylistSelect playlists={this.props.playlistData.lists}
+            selectedPlaylist={this.getCurrentPlaylist()}
```

Comment:
Hmm, so I'm no the biggest fan of all the prop drilling we have to do (I know we do it everywhere). In this case, are we passing everything in so we can set up a storyboard.

It's fine as is, but I wonder if its worth just directly using `getPlaylistAPI()` from inside the `PlaylistSelect` component? @petemill for your thoughts

Reply:
(don't feel like you need to change it)